tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -407,3 +419,37 @@ Get User Notebook PVC Name
     ${safe_username} =   Get Safe Username    ${username}
     ${notebook_pod_name} =   Set Variable  jupyterhub-nb-${safe_username}-pvc
     [Return]    ${notebook_pod_name}
+
+Clone Repo
+    [Documentation]    Clones git repo
+    [Arguments]    ${REPO_URL}
```

Comment:
Arguments should not be in capital letters

Reply:
Done